README.md

```diff
@@ -32,6 +32,17 @@ The following keybindings are available:
 - `RET` or `v`: Go to the entry at point
 - `q`: Quit pass
 
+#### 2FA / OTP Support
+
+If you have the [`pass-otp`](https://github.com/tadfisher/pass-otp) extension
+installed, you will be able to use the following keybindings as well:
+
+- `o i`: Insert an OTP key URI in a new entry
+- `o a`: Append an OTP key URI to an existing entry
```

Comment:
Say you have an entry for your Slack account, and you decide to enable 2FA in that account. You might want to store the OTP URI into the same entry where you have your Slack password instead of creating a different entry for it. That's the way I use it, anyway, but both options are available.

Reply:
I understood `"append"` as "add a second or more URI to". You might to explain what happens if there already is an URI in the entry.

Reply:
I completely agree with you, I actually had that same understanding when I started to use `pass-otp`. I added a note to the documentation trying to clarify things a little bit more.